clarity/src/vm/clarity.rs

```diff
@@ -284,6 +293,7 @@ pub trait TransactionConnection: ClarityConnection {
     /// abort_call_back is called with an AssetMap and a ClarityDatabase reference,
     ///   if abort_call_back returns true, all modifications from this transaction will be rolled back.
     ///      otherwise, they will be committed (though they may later be rolled back if the block itself is rolled back).
+    #[allow(clippy::too_many_arguments)]
     fn run_contract_call<F>(
```

Comment:
I think this time tracking should also be applied to `initialize_smart_contract`. Note that this will require some changes to your integration test as well.

Reply:
yep, i added a contract that has an explicit call (a dummy `+ 1 1`) to trigger the eval.